src/helpers/validate.js

```diff
@@ -9,7 +9,7 @@ function validateURL(url) {
 }
 
 function validateFlags(flags) {
```

Comment:
Should this be renamed to `validateFlag` (singular) since it validates *one* flag?

Reply:
Probably , but that's scope creep imo.